src/libraries/System.Private.CoreLib/src/System/BitConverter.cs

```diff
@@ -499,5 +499,15 @@ public static unsafe float Int32BitsToSingle(int value)
 
             return *((float*)&value);
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static unsafe short HalfToInt16Bits(Half value)
+        {
+            return *((short*)&value);
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static unsafe Half Int16BitsToHalf(short value)
```

Comment:
Nit: Use the same same style for both new methods in this file (e.g. either `=>` for both; or `{ }` for both)